extensions/wrapper/wrapper/src/main/java/de/sovity/edc/ext/wrapper/api/ui/UiResource.java

```diff
@@ -73,4 +76,11 @@ public TransferHistoryPage transferHistoryPageEndpoint() {
     public AssetDto getTransferProcessAsset(@PathParam("transferProcessId") String transferProcessId) {
         return transferHistoryPageAssetFetcherService.getAssetForTransferHistoryPage(transferProcessId);
     }
+
+    @GET
+    @Path("pages/contract-definition-page")
+    @Produces(MediaType.APPLICATION_JSON)
+    public ContractDefinitionPage contractDefinitionPageEndpoint() {
```

Comment:
Maybe the method name should only be contractDefinitionPage? If the above names are with the Endpoint suffix I think we should remove them as these function names are passed into the generated API client libraries